<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QPushButton" name="btn_insertar">
       <property name="text">
        <string>Insertar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_modificar">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_eliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_cerrar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QTableWidget" name="tabla_viviendas">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="11" column="0" colspan="5">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Nro Edificio</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Calle</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="9">
      <widget class="QLineEdit" name="valor_direccion_calle"/>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Piso</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="valor_piso"/>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Reparto</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="5">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fecha de inicio</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6" colspan="4">
      <widget class="QDateEdit" name="valor_fecha"/>
     </item>
     <item row="5" column="0" colspan="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Direccion</string>
       </property>
      </widget>
     </item>
     <item row="9" column="8">
      <widget class="QCheckBox" name="valor_balcon">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="7">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Balcon</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="7">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Numero de indentificacion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <widget class="QLineEdit" name="valor_numero_identificacion"/>
     </item>
     <item row="0" column="8">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Estado de terminacion</string>
       </property>
      </widget>
     </item>
     <item row="0" column="9">
      <widget class="QLineEdit" name="valor_estado_terminacion"/>
     </item>
     <item row="3" column="0" colspan="7">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cant. Metros Cuadrados</string>
       </property>
      </widget>
     </item>
     <item row="3" column="7">
      <widget class="QLineEdit" name="valor_cantidad_metro_cuadrados"/>
     </item>
     <item row="3" column="8">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Cantidad de cuartos</string>
       </property>
      </widget>
     </item>
     <item row="3" column="9">
      <widget class="QLineEdit" name="valor_cantidad_cuartos"/>
     </item>
     <item row="6" column="8">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Numero</string>
       </property>
      </widget>
     </item>
     <item row="6" column="9">
      <widget class="QLineEdit" name="valor_direccion_numero"/>
     </item>
     <item row="11" column="5">
      <widget class="QLineEdit" name="valor_numero_edificio"/>
     </item>
     <item row="11" column="7">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Cantidad de Pisos</string>
       </property>
      </widget>
     </item>
     <item row="11" column="8" colspan="2">
      <widget class="QLineEdit" name="valor_cantidad_pisos"/>
     </item>
     <item row="6" column="5" colspan="3">
      <widget class="QLineEdit" name="valor_direccion_reparto"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
